src/mscorlib/model.xml

```diff
@@ -683,6 +683,9 @@
       <Member Name="GetHashCode(T)" />
       <Member MemberType="Property" Name="Default" />
     </Type>
+    <Type Name="System.Collections.Generic.EnumEqualityComparer&lt;T&gt;">
```

Comment:
> Since the serialization is not cross-version compatible in .NET Core, do we want to remove this quirk?

Sounds fine.  I'll open an issue to do it separately.